app/views/admin/user_detail.html.haml

```diff
@@ -379,3 +386,17 @@
             = text_field_tag :reject_user_name, "", class: "form-control", placeholder: "User to delete", data: { "1p_ignore" =>  true }
             = hidden_field_tag :reject_user_id
           = submit_tag "Merge Into #{@display_user.login}", class: "btn btn-danger", data: { confirm: "This will delete the user you just chose and move all their content over to #{@display_user.login}. This can't be undone. Proceed?" }
+#replaceusername.modal.fade
+  .modal-dialog{ role: "document" }
+    .modal-content
+      = form_tag replace_username_admin_user_path(@display_user), method: :put do
+        .modal-header
+          %button.close{ type: "button", "data-dismiss" => "modal" } ×
+          %h4.modal-title= t(:moderator_actions_replaceUsername)
+        .modal-body
+          %p
+            = t(:replace_user_name_admin_explanation)
```

Comment:
Nothing on the admin page needs to be localized, so don't bother making this translatable. This just adds work for the volunteer translators that no one will ever see.